pallets/gear-program/src/tests.rs

```diff
@@ -115,7 +115,7 @@ fn pause_program_works() {
 #[test]
 fn pause_program_twice_fails() {
     new_test_ext().execute_with(|| {
-        let raw_code = hex!("0061736d01000000020f0103656e76066d656d6f7279020001").to_vec();
+        let raw_code = hex!("0061736d01000000010401600000020f0103656e76066d656d6f727902000103020100070a010668616e646c6500000a040102000b0019046e616d650203010000060d01000a656e762e6d656d6f7279").to_vec();
```

Comment:
why do the code size increased so much?

Reply:
cuz of import/export section

Reply:
shouldn't we provide here just code and append export sections in `code::new()`?

Reply:
didn't understand what you mean